test/support/factory.ex

```diff
@@ -1,6 +1,4 @@
 defmodule Crawler.Factory do
   use ExMachina.Ecto, repo: Crawler.Repo
-
-  # Define your factories in /test/factories and declare it here,
-  # eg: `use Crawler.Accounts.UserFactory`
+  use Crawler.Accounts.UserFactory
```

Comment:
How about add a break line between `use ExMachina.Ecto, repo: Crawler.Repo` and our factory list

```suggestion

  use Crawler.Accounts.UserFactory
```